README.md

```diff
@@ -1 +1,49 @@
 
+## Summary
+Humor Me is a web application that encourages users to laugh and feel light through the generation of random quotes. The user has the option to generate quotes in either a wholesome or chaotic category to best fit their modd in the moment. Additional features include allowing the user to favorite quote posters and create their own. 
+
+## Viewing the Application
+(View our app here)[https://humor-me-ui.vercel.app/]
+
+OR
+
+1. Clone down the repo to your machine using `git clone`
+2. `cd` into the directory 
+3. run `npm install`
+4. run `npm start`
+
+Happy viewing! 
+
+## Preview of Application
+- add gif of user randomly generating a wholesome quote, then a chaotic one
+
+## Context
+This project was completed during Mod 3 of Turing. The goal of this project was to learn and immplement a new technology. We chose to build a backend server (linked [here](https://github.com/rachelsoae/stretch-be)) to host our quotes, and images, as well as save any user favorited posters. 
+
+## Contributors
+- [Jamie Caudill](https://github.com/JamieCaudill)
+- [Daniel Cochico](https://github.com/dcochico)
+- [Rachel Soae Prather](https://github.com/rachelsoae)
+- [Alexa VanSchaardenburg](https://github.com/AlexaVanSchaardenburg)
+
+## Technologies Used
+Front End Tech Stack
+- React.js
+- React Router
+- Cypress
+- CSS
+
+Back End Tech Stack
+- Express.js
+- Knex
+- Postgres
+
+## Goals and Achievements
+
+## Future Extensions
+Given more time on this porject or upon revisting this project, extensions include: 
+- Adding delete functionality to allow a user to delete favorited posters
```

Comment:
I would probably remove the parts of these that say "Adding x functionality to..." and justlead with the user story